src-plugins/itkDataImage/interactors/medVtkViewItkDataImage4DInteractor.cpp

```diff
@@ -169,6 +172,8 @@ void medVtkViewItkDataImage4DInteractor::setInputData(medAbstractData *data)
             d->view2d->SetColorRange(range);
             this->initWindowLevelParameters(range);
 
+            medVtkViewItkDataImageInteractor::createSlicingParam();
```

Comment:
Using a fully-qualified function call here is not needed and should be avoided.

Reply:
It's to avoid duplication, it's not the same class. Is it ok?

Reply:
Calling ```createSlicingParam()``` is sufficient here, wherever the function is. Qualified function names are used when inside a derived virtual function and you want to specifically call the same function in a parent class (like we do for some of the toolbox ```clear``` functions). Outside that sort of use, it is not needed and can create unexpected bugs (for example, if someone decides to write a version of ```createSlicingParam``` specific for ```medVtkViewItkDataImage4DInteractor```, they would make the base function virtual, add an override, and expect it to be called, but it wouldn't).

Reply:
@fcollot done